### ISSUE TITLE
extraction error <ref *1> Error: spawnSync pandoc-citeproc ENOENT

### ISSUE THREAD

**Issue by methridge** [phiresky/pandoc-url2cite#5]
Pandoc 2.11 has removed the `pandoc-citeproc` executable which `pandoc-url2cite` is spawning a process to use, thus causing all citations to fail.

```
pandoc 2.11.1
Compiled with pandoc-types 1.22, texmath 0.12.0.3, skylighting 0.10.0.3,
citeproc 0.1.0.3, ipynb 0.1.0.1
```

```
fetching citation from url https://www......
extraction error <ref *1> Error: spawnSync pandoc-citeproc ENOENT
    at Object.spawnSync (node:internal/child_process:1074:20)
    at spawnSync (node:child_process:622:24)
    at Object.execFileSync (node:child_process:649:15)
    at bibtex2csl (/usr/local/lib/node_modules/pandoc-url2cite/dist/index.js:33:33)
    at getCslForUrl (/usr/local/lib/node_modules/pandoc-url2cite/dist/index.js:53:25)
    at processTicksAndRejections (node:internal/process/task_queues:93:5)
    at async Url2Cite.getCslForUrlCached (/usr/local/lib/node_modules/pandoc-url2cite/dist/index.js:191:32)
    at async Object.Url2Cite.astTransformer [as single] (/usr/local/lib/node_modules/pandoc-url2cite/dist/index.js:148:21)
    at async walk (/usr/local/lib/node_modules/pandoc-url2cite/node_modules/pandoc-filter/index.js:60:28)
    at async walk (/usr/local/lib/node_modules/pandoc-url2cite/node_modules/pandoc-filter/index.js:79:22) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'spawnSync pandoc-citeproc',
  path: 'pandoc-citeproc',
  spawnargs: [ '--bib2json', '--format=biblatex' ],
  error: [Circular *1],
  status: null,
  signal: null,
  output: null,
  pid: 89825,
  stdout: null,
  stderr: null
}
Error running filter pandoc-url2cite:
Filter returned error status 1
```

**Comment by phiresky** [comment]
This should have already been fixed with #4 (version 0.6.6). How did you install pandoc-url2cite? Try updating it by running npm install -g pandoc-url2cite again.

**Comment by methridge** [comment]
🤦 My apologies.  I found an old version buried in an old path that wasn't being updated.
